ios/Runner/AppDelegate.swift

```diff
@@ -114,6 +123,19 @@ import flutter_local_notifications
         }
         return false
     }
+    
+    private func handleRemoteNotificationData(userInfo: [AnyHashable : Any]) {
+        guard let emailId = userInfo[JmapConstants.EMAIL_ID] as? String,
+              !emailId.isEmpty else {
+            return
+        }
+        TwakeLogger.shared.log(message: "handleRemoteNotificationData:emailId = \(emailId)")
+        self.saveEmailNotificationId(emailId: emailId)
+    }
+    
+    private func saveEmailNotificationId(emailId: String) {
+        UserDefaults.standard.set(emailId, forKey: CoreUtils.IOS_EMAIL_NOTIFICATION_ID)
+    }
```

Comment:
maybe we need try to store also the time of `click_event`. it will help us, not open the wrong email :)

Reply:
I don't understand what time has to do with this. IMO, we don't need it this way.